docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/config-based-source.md

```diff
@@ -0,0 +1,462 @@
+---
+id: config-based-source
+title: Config Based Source
+sidebar_label: STIX/TAXII 2 Client
+tags:
+  - cloud-to-cloud
+  - config
+  - based
+description: Learn how to set up a Config Based C2C to collect data into the Sumo Logic environment.
+---
```

Comment:
I've noticed the title does not appear and I don't know why.

<img width="1597" alt="image" src="https://github.com/SumoLogic/sumologic-documentation/assets/45008325/116dca0a-5786-45ca-9d6f-6de71fe10a25">

Reply:
It looks like the author used one hash on Setup, which markdown reads as a title, and then itoverrides the actual title. So you just have to add a second hash and that'll fix it.

<img width="350" alt="Screenshot 2024-03-07 at 1 43 09 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/0357296c-72fd-48c0-bdd0-13dfb47ee06c">

Reply:
fixed